handlers/facebookapp/facebookapp_test.go

```diff
@@ -502,50 +785,82 @@ func buildMockFBGraph(testCases []ChannelHandleTestCase) *httptest.Server {
 
 		// user has a name
 		if strings.HasSuffix(r.URL.Path, "1337") {
-			w.Write([]byte(`{ "first_name": "John", "last_name": "Doe"}`))
+			w.Write([]byte(`{ "name": "John Doe"}`))
 			return
 		}
 
 		// no name
-		w.Write([]byte(`{ "first_name": "", "last_name": ""}`))
+		w.Write([]byte(`{ "name": ""}`))
 	}))
 	graphURL = server.URL
 
 	return server
 }
 
 func TestDescribe(t *testing.T) {
```

Comment:
I think it is best to have 2 separate tests one for FBA and the other for IG instead of the for loop on a slice of slices and the if condition